pkg/rc/replication_controller.go

```diff
@@ -498,11 +503,7 @@ func (rc *replicationController) checkForIneligible(current types.PodLocations,
 	}
 
 	if len(ineligibleCurrent) > 0 {
-		errMsg := fmt.Sprintf("RC has scheduled %d ineligible nodes: %s", len(ineligibleCurrent), ineligibleCurrent)
-		err := rc.alerter.Alert(rc.alertInfo(errMsg))
-		if err != nil {
-			rc.logger.WithError(err).Errorln("Unable to send alert")
-		}
+		rc.transferNodes(ineligibleCurrent)
```

Comment:
`errcheck` in the build noticed that you're not checking the error that `transferNodes()` returns here

Reply:
I found this by running `errcheck -ignoretests github.com/square/p2/pkg/... | grep -v defer` which is the line that failed in the build (but you have to unwrap it from the `exit() | wc -l`)